.github/workflows/verify.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
           ref: ${{ github.event.pull_request.head.ref }}
```

Comment:
I believe this should fix the original CI issues you hit and ensure that the merge commit is being tested against the current repo instead of the user's fork:

```suggestion
```

Reply:
Let's find out!